packages/cashier/src/stores/__tests__/account-transfer-store.spec.js

```diff
@@ -430,10 +430,10 @@ describe('AccountTransferStore', () => {
         expect(account_transfer_store.accounts_list[0].text).toMatch(/^Deriv X(.)*$/);
         expect(account_transfer_store.accounts_list[1].text).toMatch(/^Deriv X(.)*$/);
         expect(account_transfer_store.accounts_list[2].text).toMatch(/^Deriv X(.)*$/);
-        expect(account_transfer_store.accounts_list[3].text).toMatch(/^DMT5(.)*$/);
-        expect(account_transfer_store.accounts_list[4].text).toMatch(/^DMT5(.)*$/);
-        expect(account_transfer_store.accounts_list[5].text).toMatch(/^DMT5(.)*$/);
-        expect(account_transfer_store.accounts_list[6].text).toMatch(/^DMT5(.)*$/);
+        expect(account_transfer_store.accounts_list[3].text).toMatch(/^(.)*$/);
```

Comment:
This shouldn't be like this:

```suggestion
        expect(account_transfer_store.accounts_list[3].text).toMatch(/^Deriv MT5(.)*$/);
```

Reply:
no, they do not need Deriv MT5 , I have removed this lines from spec as we are testing only sorting functionality here